core/players-handbook-2024/classes/class-fighter.xml

```diff
@@ -0,0 +1,337 @@
+			<description level="20">You can attack four times, instead of once, whenever you take the Attack action on your turn</description>
+		</sheet>
+		<rules>
+			<stat name="extra attack:count" value="2" level="5" bonus="extra attack" />
+			<stat name="extra attack:count" value="3" level="11" bonus="extra attack" />
+			<stat name="extra attack:count" value="4" level="20" bonus="extra attack" />
+		</rules>
+	</element>
+
+	<element name="Level 5: Tactical Shift" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_TACTICAL_SHIFT">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_TACTICAL_SHIFT</requirements>
+		<description>
+			<p>Whenever you activate your Second Wind with a Bonus Action, you can move up to half your speed without provoking Opportunity Attacks.</p>
+		</description>
+		<sheet alt="Tactical Shift">
+			<description>When you activate Second Wind with a Bonus Action, you can move up to half your speed without provoking Opportunity Attacks.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 9: Indomitable" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_INDOMITABLE">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_INDOMITABLE</requirements>
+		<description>
+			<p>If you fail a saving throw, you can reroll it with a bonus equal to your Fighter level. You must use the new roll, and you can't use this feature again until you finish a Long Rest.</p>
+			<p class="indent">You can use this feature twice between long rests starting at 13th level and three times between long rests starting at 17th level.</p>
+		</description>
+		<sheet alt="Indomitable" usage="{{indomitable:usage}}/Long Rest">
+			<description>You can reroll a saving throw that you fail with a +{{indomitable:bonus}} bonus. If you do so, you must use the new roll.</description>
+		</sheet>
+		<rules>
+			<stat name="indomitable:usage" value="1" level="9" />
+			<stat name="indomitable:usage" value="1" level="13" />
+			<stat name="indomitable:usage" value="1" level="17" />
+			<stat name="indomitable:bonus" value="level:fighter" />
+		</rules>
+	</element>
+
+	<element name="Level 9: Tactical Master" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_TACTICAL_MASTER">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_TACTICAL_MASTER</requirements>
+		<description>
+			<p>When you attack with a weapon whose mastery property you can use, you can replace that property with the Push, Sap or Slow property for that attack.</p>
+		</description>
+		<sheet alt="Tactical Master">
+			<description>When you attack with a weapon whose mastery property you can use, you can replace that property with the Push, Sap or Slow property for that attack.</description>
+		</sheet>
+	</element>
+
```

Comment:
```suggestion

	<element name="Level 11: Two Extra Attacks" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_FIGHTER_TWO_EXTRA_ATTACKS">
		<requirements>!(ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_EXTRA_ATTACK||ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_FIGHTER_TWO_EXTRA_ATTACKS)</requirements>
		<description>
			<p>You can attack three times instead of once whenever you take the Attack action on your turn.</p>
		</description>
		<sheet display="false" />
		<rules>
			<stat name="extra attack:count" value="3" bonus="extra attack" />
		</rules>
	</element>

```